python/cudnn/gemm/frost/kernel_templates/sm100_block_scale_matmul_2ctamma.py

```
@@ -937,131 +954,139 @@ def _kernel(
-                )
+                    cute.arch.fence_view_async_shared()
+                    nvvm.barrier_cta_sync(
+                        barrier_id=EPI_SYNC_BAR_ID,
+                        thread_count=num_epilogue_warps * 32,
+                    )
 
-                if warp_idx == 0:
-                    if cutlass.const_expr(cd_out_is_m_major):
-                        for _mb in cutlass.range_constexpr(cta_tile_mnk[0] // cd_mmajor_atom_m):
+                    if warp_idx == 0:
+                        if cutlass.const_expr(cd_out_is_m_major):
```

Ruff (0.16.1)

[error] Undefined name `cd_out_is_m_major`

(F821)